src/crud.py

```
@@ -30,10 +34,115 @@
+        """
+
+        batches: list[tuple[list[str], list[str]]] = []
+        current_batch_ids: list[str] = []
+        current_batch_texts: list[str] = []
+        current_batch_tokens = 0
+
+        # OpenAI's official limits for embeddings API:
+        # - Max 300,000 tokens per request (total across all inputs)
+        # - Max 8,192 tokens per individual input
+        # Note: Schema limits ensure we'll never exceed the 2,048 inputs per request limit
+        MAX_TOKENS_PER_REQUEST = settings.LLM.MAX_EMBEDDING_TOKENS_PER_REQUEST
```

Pylint (3.3.7)

[convention] Variable name "MAX_TOKENS_PER_REQUEST" doesn't conform to snake_case naming style

(C0103)